test/parallel/test-cluster-bind-privileged-port.js

```diff
@@ -21,6 +21,19 @@
 
 'use strict';
 const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const net = require('net');
+
+const { execSync } = require('child_process');
+const sysctlOutput = execSync('sysctl net.ipv4.ip_unprivileged_port_start').toString();
+const unprivilegedPortStart = parseInt(sysctlOutput.split(' ')[1], 10);
```

Comment:
This should be inside the `if (common.isLinux) {` block because it won't work on other platforms.

Index 1 is the `=` character, not the port number.